frontend/.env.uat

```
@@ -0,0 +1,3 @@
+VITE_API_BASE_URL=https://api-uat.fairworkly.com/api
+VITE_APP_ENV=uat
+VITE_DEBUG=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)